basic-cradle-manager/src/main/java/com/exactpro/cradle/module/DummyCradleManager.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020-2022 Exactpro (Exactpro Systems Limited)
```

Comment:
```suggestion
 * Copyright 2022 Exactpro (Exactpro Systems Limited)
```
The first date should be year of file made